code/modules/food&drinks/food/snacks_meat.dm

```diff
@@ -109,6 +109,22 @@
 	desc = "A human meat, on a stick."
 	bonus_reagents = list("nutriment" = 1, "vitamin" = 6)
 
+/obj/item/weapon/reagent_containers/food/snacks/kebab/human/lizard
+	name = "lizard-kebab"
+	desc = "A lizard meat, on a stick."
```

Comment:
Get rid of the "A" in front of all of the kebab descriptions, including the human one. It reads like broken english.